### DISCUSSION TITLE
Can you disable the background blur in song selection?

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#13687] [Q&A] [answered]
I don't like how the background is blurred. Makes it feel congested for lack of a better word. If you can't, will/can it be implemented?

**Comment by peppy** [comment, marked as answer]
This isn't possible right now. I'm not sure if we will offer this, depends on how many users dislike it I guess.

**Reply by NafeeJ** [reply]
ok, hope it will be.

**Comment by JDECOMBE** [comment]
Don't know how feasible it is to do a year and a half later, so bumping this idea with some pictures since this is the one thing that prevents me from switching to lazer. I'm mostly using the background image to recognize a map and having it blurred isn't ideal for me. Having the option to toggle it would be great !

# How it is currently:
![image](https://user-images.githubusercontent.com/25856127/213884836-78492559-5298-48ba-a153-662437ae70fc.png)

# How I'm imagining it with the blur disabled
![image](https://user-images.githubusercontent.com/25856127/213884840-3053fb03-935a-4790-9eec-598fa205b195.png)

https://user-images.githubusercontent.com/25856127/213927027-b34933b4-a6ef-4a84-87c3-43fbc146bbf2.mp4

**Reply by peppy** [reply]
The hope was to add this as a setting in the skin layout editor, but I'm not against to it being done in the main game settings for now to ease adoption. It will require some careful consideration as there's already a lot of different components interacting with the beatmap background screen to adjust these kinds of settings.

**Reply by JDECOMBE** [reply]
Nice ! I'll try to work on that as soon as possible (possibly today) and make a PR. Is that okay ?

**Reply by peppy** [reply]
You're welcome to give it a shot. I know I have a branch somewhere that did this but it's quite old and likely easier to start from scratch.

**Reply by Inconsist** [reply]
Want to add that I also heavily rely on visuals to recognize beatmaps and love the idea. Ideally when switching though beatmaps quickly it would also be possible to load the backgounds faster, maybe at a lower resolution at first to speed it up.
